README.md

```diff
@@ -1,75 +1,92 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Rails project: Budget App
 
-> Description the project.
+> Budget calculator website, which has categories of budgets in which you can enter transaction costs for each expense in other to get the total budget price in that particular budget category.
 
+## Screenshot
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+## Link to video
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+## Link to App
```

Comment:
- [ ] It would be better if we add the link to the deployed version of the app. The app should be deployed on Heroku, netlify or gh-pges

Reply:
Hi @addod19, a question... is it OK if I deploy the current branch version once I still do not have my project approved in order to merge it to main and then deploy? Thanks in advance.

Reply:
It is also done!